src/Symfony/Component/Filesystem/Tests/FilesystemTest.php

```diff
@@ -878,6 +879,17 @@ public function providePathsForMakePathRelative()
             array('C:/aa/bb/../../cc', 'C:/aa/../dd/..', 'cc/'),
             array('C:/../aa/bb/cc', 'C:/aa/dd/..', 'bb/cc/'),
             array('C:/../../aa/../bb/cc', 'C:/aa/dd/..', '../bb/cc/'),
+            array('aa/bb', 'aa/cc', '../bb/'),
```

Comment:
@chalasr Actually, the example that you gave was a test I mistakenly added in that very same PR. So it was never part of a merge. :)